apps/web/src/services/asset-inventory/collector/modules/CollectorItemInfo.vue

```diff
@@ -0,0 +1,236 @@
+            />
+            <p-button style-type="transparent">
+                <p-i v-if="state.toggleActive"
+                     name="ic_edit"
+                     height="0.75rem"
+                     width="0.75rem"
+                     color="inherit"
+                     class="icon-schedule"
+                />
+                <p-i v-else
+                     name="ic_settings-filled"
+                     height="0.75rem"
+                     width="0.75rem"
+                     color="inherit"
+                     class="icon-schedule"
+                />
+                {{ state.toggleActive ? 'Edit' : 'Set' }} Schedule
+            </p-button>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { reactive } from 'vue';
+
+import {
+    PButton, PI, PLazyImg, PToggleButton,
+} from '@spaceone/design-system';
+
+import type { CollectorModel } from '@/services/asset-inventory/collector/type';
+import { CollectorItemInfoType } from '@/services/asset-inventory/collector/type';
+
+interface Props {
+    label: string;
+    item?: CollectorModel;
+    type: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+    label: '',
+    item: undefined,
+    type: '',
+});
+
+
+const state = reactive({
+    toggleActive: false,
+    toggleStatus: 'OFF',
+});
+
+/* Components */
+const handleChangeToggle = () => {
+    state.toggleActive = !state.toggleActive;
+    state.toggleStatus = state.toggleActive ? 'ON' : 'OFF';
+};
+</script>
+
+<style scoped lang="postcss">
+.collector-item-info {
+    .info-item {
+        @apply flex flex-col;
+        gap: 0.5rem;
+
+        .info-label {
+            @apply text-label-sm text-gray-500;
+        }
+
+        .plugin {
+            @apply flex items-center text-label-md;
+            gap: 0.25rem;
+
+            .plugin-name {
+                @apply truncate;
+                flex: 1;
+                max-width: 11rem;
+            }
+
+            .plugin-version {
+                @apply text-label-sm text-gray-700;
+            }
+        }
+
+        .jobs-wrapper {
+            @apply flex;
+            gap: 0.375rem;
+
+            .icon-fill-wrapper {
+                @apply relative bg-green-600 rounded;
+                width: 1rem;
+                height: 1rem;
+
+                .check-icon {
+                    @apply absolute text-white;
+                    left: 50%;
+                    top: 50%;
+                    transform: translate(-35%, -50%);
+                }
+            }
+        }
+
+        .to-history-detail {
+            @apply text-label-sm text-blue-700;
+        }
+
+        /* custom design-system component - p-toggle-button */
+        :deep(.p-toggle-button) {
+            .label {
+                @apply text-gray-400;
+            }
+            &.toggle-active {
+                .label {
+                    @apply text-blue-600;
+                }
+            }
+        }
```

Comment:
suggestion: I think in this case, I don't think using `label` prop is ideal.
I don't know why, but in the figma, there is no label but it is in storybook. 🤔
It seems inconsistency between the figma and the storybook, so we must talk about the `label` prop with the designers first, instead of deeply changing css of mirinae component.

FYI
We avoid changing mirinae component's css deeply. That's because it causes lot of refactoring when the mrinae component's markup hierarchy was changed.